src/foremast/s3/s3apps.py

```diff
@@ -30,33 +30,37 @@
 class S3Apps(object):
     """Configure infrastructure and policies for S3 web applications."""
 
-    def __init__(self, app, env, region, prop_path):
+    def __init__(self, app, env, region, prop_path, primary_region='us-east-1'):
         """S3 application object. Setups Bucket and policies for S3 applications.
 
         Args:
             app (str): Application name
             env (str): Environment/Account
             region (str): AWS Region
             prop_path (str): Path of environment property file
+            primary_region (str): The primary region for the application.
```

Comment:
Same as above, default_region makes it a bit more like AWS CLI (in that you can specify a default region if none specified)

Reply:
Disregard!